src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -18,11 +18,22 @@ static ucs_config_field_t uct_rdmacm_md_config_table[] = {
 
   {"ADDR_RESOLVE_TIMEOUT", "500ms",
    "Time to wait for address resolution to complete",
-    ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
+   ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
 
   {NULL}
 };
 
+static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
```

Comment:
move to the rdmacm_cm.c file please. this file is approaching its EOL :)

Reply:
no way, component is registered here

Reply:
so..?
uct_rdmacm_cm_config_t is defined in rdmacm_cm.h, it would still work.

Reply:
no, `uct_rdmacm_cm_config_table` should be defined in *.c as well as component to avoid global `external` variable

Reply:
we already have an external param in uct_cm.h for uct_cm_config_table[], don't see why not do it here as well.
i don't thinks it's a good idea to keep adding code to a file we plan to remove if it's possible not to.

Reply:
I don't think we can remove this file easily, likely just cleanup and rename

Reply:
> we already have an external param in uct_cm.h for uct_cm_config_table[]

I would try to avoid this...